From weiyi.huang@gmail.com  Wed Nov 12 14:55:20 2008
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Fri, 31 Oct 2008 22:49:56 +0800
Subject: Staging: wlan-ng: remove unused #include <version.h>
To: gregkh@suse.de
Message-ID: <20081031223057.0F1A.WEIYI.HUANG@gmail.com>


The file(s) below do not use LINUX_VERSION_CODE nor KERNEL_VERSION.
  drivers/staging/wlan-ng/p80211conv.c
  drivers/staging/wlan-ng/p80211req.c
  drivers/staging/wlan-ng/p80211wext.c
  drivers/staging/wlan-ng/prism2mgmt.c

This patch removes the said #include <version.h>.

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/p80211conv.c |    1 -
 drivers/staging/wlan-ng/p80211req.c  |    1 -
 drivers/staging/wlan-ng/p80211wext.c |    1 -
 drivers/staging/wlan-ng/prism2mgmt.c |    1 -
 4 files changed, 4 deletions(-)

--- a/drivers/staging/wlan-ng/p80211conv.c
+++ b/drivers/staging/wlan-ng/p80211conv.c
@@ -52,7 +52,6 @@
 /*================================================================*/
 /* System Includes */
 
-#include <linux/version.h>
 
 #include <linux/module.h>
 #include <linux/kernel.h>
--- a/drivers/staging/wlan-ng/p80211req.c
+++ b/drivers/staging/wlan-ng/p80211req.c
@@ -54,7 +54,6 @@
 /* System Includes */
 
 
-#include <linux/version.h>
 
 #include <linux/module.h>
 #include <linux/kernel.h>
--- a/drivers/staging/wlan-ng/p80211wext.c
+++ b/drivers/staging/wlan-ng/p80211wext.c
@@ -38,7 +38,6 @@
 /* System Includes */
 
 
-#include <linux/version.h>
 
 #include <linux/kernel.h>
 #include <linux/sched.h>
--- a/drivers/staging/wlan-ng/prism2mgmt.c
+++ b/drivers/staging/wlan-ng/prism2mgmt.c
@@ -61,7 +61,6 @@
 /* System Includes */
 #define WLAN_DBVAR	prism2_debug
 
-#include <linux/version.h>
 
 #include <linux/if_arp.h>
 #include <linux/module.h>
